python/tchannel/scheme.py

```diff
@@ -77,3 +77,25 @@ def deserialize_body(self, obj):
 
     def serialize_body(self, obj):
         return json.dumps(obj)
+
+
+class ThriftArgScheme(ArgScheme):
+    def __init__(self):
+        self._type = "thrift"
+
+    def type(self):
+        return self._type
```

Comment:
Why assign to this attribute at all? This will never change. Just make it a class attribute with a constant value and return that.